lenskit-cli/src/main/java/org/lenskit/cli/util/Logging.java

```diff
@@ -72,10 +73,15 @@ public static void configureLogging(Namespace options) {
         if (System.getProperty("logback.configurationFile") != null) {
             return;
         }
-        boolean debug = options.getBoolean("debug");
         boolean debugGrapht = options.getBoolean("debug_grapht");
         File logFile = options.get("log_file");
 
+        String lstr = options.getString("log_level");
+        Level logLevel = Level.toLevel(lstr, Level.INFO);
+        String lfstr = options.getString("log_file_level");
+        Level logFileLevel;
```

Comment:
no having logFileLevel declared and then set on the next line, instead of keeping it on one line like you do for logLevel.